pkg/internal/tracking/context.go

```diff
@@ -8,7 +8,7 @@ import (
 )
 
 const (
-	ProviderVersion string = "v0.99.0" // TODO(SNOW-1814934): Currently hardcoded, make it computed
+	ProviderVersion string = "v1.0.0-rc.1" // TODO(SNOW-1814934): Currently hardcoded, make it computed
```

Comment:
make it `v1.0.0-rc1`

Reply:
I initially did, but realized that in semantic versioning they use `.` here: https://semver.org/

Reply:
`A pre-release version MAY be denoted by appending a hyphen and a series of dot separated identifiers immediately following the patch version.` (https://semver.org/#spec-item-9) - so in our case I would say that rc1 is and identifier, so I agree with @sfc-gh-jcieslak

Reply:
I'll change this in the next pr.